client/flags/flags.go

```diff
@@ -97,7 +97,7 @@ func AddTxFlagsToCmd(cmd *cobra.Command) {
 	cmd.Flags().String(FlagFrom, "", "Name or address of private key with which to sign")
 	cmd.Flags().Uint64P(FlagAccountNumber, "a", 0, "The account number of the signing account (offline mode only)")
 	cmd.Flags().Uint64P(FlagSequence, "s", 0, "The sequence number of the signing account (offline mode only)")
-	cmd.Flags().String(FlagMemo, "", "Memo to send along with transaction")
+	cmd.Flags().String(FlagNote, "", "Note to add a description to the transaction")
```

Comment:
```suggestion
	cmd.Flags().String(FlagNote, "", "Note to add a description to the transaction (previously `--memo`)")
```

as per https://github.com/cosmos/cosmos-sdk/issues/9122#issuecomment-822370202